site/docs/v1/tech/apis/jwt.adoc

```diff
@@ -308,7 +311,11 @@ include::docs/src/json/jwt/refresh-post-response.json[]
 
 [.api]
 [field]#access_token# [type]#[String]#::
-The encoded access token. This cookie is written in the response as an HTTP Only session cookie.
+The encoded access token. This cookie is written in the response as an `HTTPOnly`, `SameSite=Lax` session cookie.
+
+[field]#refresh_token# [type]#[String]#::
+The encoded refresh token. This cookie is written in the response as an `HTTPOnly`, `SameSite=Lax` session cookie.
```

Comment:
The refresh token cookie is not written as a session cookie. It is written persistent based upon the configured TTL.

Reply:
That's not the behavior I'm seeing. On calling `/api/jwt/refresh`, I see 

```
 Set-Cookie: refresh_token=4_...; Path=/; HttpOnly; SameSite=Lax
```

When I call 
`/api/login` I do see the refresh_token cookie with a persistent value:

```
Set-Cookie: refresh_token=4_...; Max-Age=2592000; Expires=Fri, 14-Aug-2020 21:17:18 GMT; Path=/; HttpOnly; SameSite=Lax
```

Should I file a bug?

Reply:
This was fixed under https://github.com/FusionAuth/fusionauth-issues/issues/726 in 1.18.0. If you still see this behavior after 1.18.0, it will be a new bug.